docs/wiki/DeploymentFlow.md

```diff
@@ -30,6 +30,17 @@ Modules in this reference implementation must be deployed in the following order
 |   9   | Corp Connected Spoke Network           | Creates Spoke networking infrastructure with Virtual Network Peering to support Hub & Spoke network topology.  Spoke subscriptions are used for deploying construction sets and workloads. | Management Groups, Hub Networking & Subscription for spoke networking  | [infra-as-code/bicep/modules/spokeNetworking](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/spokeNetworking)                               |
 
+## Other Deployment Scenarios
+
+The following modules can be used to deploy components representing different scenarios according to the Azure Landing Zone conceptual architecture.
+
+### Scenario: On-premises connectivity using Azure vWAN
+
+| Order | Module                                 | Description                                                                                                                                                                                | Prerequisites                                                          | Module Documentation                                                                                                                                                  |
+| :---: | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|   1   | Virtual WAN Connectivity                      | Deploys the Virtual WAN network topology and its components according to the Azure Landing Zone conceptual architecture.                                                                                          | Management Groups, Subscription for vWAN connectivity.                    | [infra-as-code/bicep/modules/vwanConnectivity](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/vwanConnectivity)                             |
+|   2   | VNet Peering with vWAN                        | Connect a virtual network to a Virtual WAN hub.                                                                                          | Management Groups, Subscription for spoke VNet, vWAN Connectivity Module                    | [TBD](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/vnetPeeringvWAN)                             |
```

Comment:
We want another shortname than TBD for the link

Reply:
since the module doesn't exist yet, I'd removed the link and renamed it to [To be developed]